src/dll/Hooks/InitScripts.cpp

```diff
@@ -0,0 +1,75 @@
+#include "InitScripts.hpp"
+#include "Addresses.hpp"
+#include "App.hpp"
+#include "Hook.hpp"
+#include "Systems/ScriptSystem.hpp"
+#include "stdafx.hpp"
+
+namespace
+{
+bool isAttached = false;
+
+void* _CBaseEngine_InitScripts(RED4ext::CBaseEngine* aEngine, unsigned __int8 a2, unsigned __int16 a3);
+Hook<decltype(&_CBaseEngine_InitScripts)> CBaseEngine_InitScripts(Addresses::CBaseEngine_InitScripts,
+                                                                  &_CBaseEngine_InitScripts);
+
+void* _CBaseEngine_InitScripts(RED4ext::CBaseEngine* aEngine, unsigned __int8 a2, unsigned __int16 a3)
+{
+    auto scriptSystem = App::Get()->GetScriptSystem();
+    auto original = aEngine->scriptsBlobPath;
+    if (aEngine->scriptsBlobPath.Length())
+    {
+        spdlog::info("Found scriptsBlobPath: '{}'", aEngine->scriptsBlobPath.c_str());
+        scriptSystem->SetUsingRedmod(true);
```

Comment:
Why is this set to true when `scriptsBlobPath` is not empty? Is RedMod doing something funny?

Reply:
My understanding of this is that `scriptsBlobPath` is only set when REDmods are being used.

Reply:
👍

Reply:
It is, but `scriptsBlobPath` was available long before redmod and works without it, so I suggest removing any mention of redmod,

Reply:
> It is, but `scriptsBlobPath` was available long before redmod and works without it, so I suggest removing any mention of redmod.

Removed this & simplified things a bit :)